### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Url Rewrite- Outbound Rules_Keyboard: After expanding macros button, user is unable to navigate through the list items using keyboard keys

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#152]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard; #MAS2.1.1

**Environment Details:**
URL: https://manage.iis.net/webserver/url-rewrite
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.1.1

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Url Rewrite and press enter
6. Tab till outbound rules tab and press enter
7. Tab till Create Rule button and press enter
8. Tab till macros and press enter. Verify user is able to navigate through the list items using keyboard keys

**Actual Result:**
User is unable to navigate through the list items using keyboard keys

**Expected Result:**
User should be able to navigate through the list items using keyboard keys

**User Impact:**
Disable users may find difficult while working with the application if focus does not go through the list items using keyboard keys

**Attachments:**
[Bug152_outbound rules_unable to navigate through the list items.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331030/Bug152_outbound.rules_unable.to.navigate.through.the.list.items.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
